MerchantSDK/GiniMerchantSDK/Sources/GiniMerchantSDK/Core/PaymentComponent/BanksBottomView.swift

```diff
@@ -191,6 +192,7 @@ extension BanksBottomView {
         static let closeIconSize = 24.0
         static let titleViewTitleIconSpacing = 10.0
         static let topAnchorPoweredByGiniConstraint = 5.0
+        static let bottomViewHeight = 44.0
```

Comment:
Maybe put this constant into common config or constants for all PaymentComponents?

Reply:
Good idea, but for now we don't have a common config. All constants are private in their views. For this one just happened to be the same in all places, but they might change